tests/app/core/models/session-model.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { assertSessionDataIntegrity } from '../../../../src/app/core/models/session.model';
+import { of } from 'rxjs';
+
+const validSessionData = {
```

Comment:
Maybe as good practice we could define the type of `validSessionData` to `validSessionData: SessionData`